.dockerignore

```diff
@@ -1,7 +1,7 @@
-.angular
-.github
-.git
-.vscode
-dist
-node_modules
-sessions
\ No newline at end of file
+/.angular
```

Comment:
undo this and leave the broader ignores

Reply:
We wouldn't want to ignore a nested directory or file named "sessions" or "dist," would we? Are you focusing on the `.angular` Docker build context exclusion, or referring to the whole list?

Reply:
I'll address this .dockerignore issue separately from this PR.